### PR TITLE
Add instruction to install on linux server edition

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related with https://github.com/omichel/webots/pull/4765.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: guide/installation-procedure.md [hunk 1]
```diff
@@ -137,6 +137,14 @@ or
 gdebi webots_{{ webots.version.major }}.{{ webots.version.minor }}.{{ webots.version.bugfix }}_amd64.deb
 ```
 
+#### Server edition
+
+Webots requires some graphical features that are usually not available by default on a linux server edition and additional packages need to be available to make it work.
+
+Webots needs the X Window system and thus the `xserver-xorg-core` and `libxslt1.1` packages have to be installed. These packages are automatically installed when using the DEB package, but in case of the tarball package the user has to manually install them.
```
Comment (omichel) [inline guide/installation-procedure.md]: Note that libxslt is not related with the X Window system, it's an XML-related library. Therefore, we shouldn't mention it here.

**Comment by stefaniapedrazzi** [comment]
:+1:

File: guide/installation-procedure.md [hunk 2]
```diff
@@ -137,6 +137,14 @@ or
 gdebi webots_{{ webots.version.major }}.{{ webots.version.minor }}.{{ webots.version.bugfix }}_amd64.deb
 ```
 
+#### Server edition
+
+Webots requires some graphical features that are usually not available by default on a linux server edition and additional packages need to be available to make it work.
+
+Webots depends on `xserver-xorg-core` and `libxslt1.1`. These packages are automatically installed when using the DEB package, but in case of the tarball package the user has to manually install them.
```
Comment (omichel) [inline guide/installation-procedure.md]: I don't understand why you mention `libxslt1.1` and not all the other dependencies... Can't we include libxslt1.1 also in the tarball, like other dependencies? Why shall we make a special case for this one and not the others?

Comment (stefaniapedrazzi) [inline guide/installation-procedure.md]: Because this is a special case and only very few user will user a server edition where these two packages are not available.
In desktop edition this packages are installed by default.
It seems a pity to make the tarball package bigger just for very few users.
But I can include them in the tarball package.

Comment (omichel) [inline guide/installation-procedure.md]: How big is it?

Comment (stefaniapedrazzi) [inline guide/installation-procedure.md]: I will check in the server edition if copying libxslt is enough or other libraries are required too.
Then I can evaluate the size.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by omichel** [comment]
:eyes:
